### DISCUSSION TITLE
Failed to auto close tag in html template since v1.77.0

### DISCUSSION THREAD

**Discussion by cWenyu** [microsoft/vscode-discussions#726] [Extension Development QnA] [answered]
Hi, community, 

I had an extension to help me recognize tags and auto close tags in `lit-html` template by using [typescript-lit-html-plugin](https://github.com/microsoft/typescript-lit-html-plugin). But auto close doesn't work anymore since vs code 1.77.0, is any new API need to be updated to enable it? Or did I miss some important updates? 

Best regards
Wenyu

**Comment by isidorn** [comment, marked as answer]
I suggest to file an issue in the extension https://github.com/microsoft/typescript-lit-html-plugin

Closing this, as this discussion is a place for extension author to ask questions.

**Comment by cWenyu** [comment]
The typescript version caused it, I was on v4.6.4, and v5.2.2 solved the issue. Just in case someone is facing the same problem.
